### DISCUSSION TITLE
Why is self.feature_extractor.nb_max_frames subtracted from total frames

### DISCUSSION THREAD

**Discussion by stri8ed** [SYSTRAN/faster-whisper#750] [Q&A] [answered]
I'm trying to understand the logic behind this subtraction: 

https://github.com/SYSTRAN/faster-whisper/blob/1eb9a8004c509a4af2960955374520c35b7b793a/faster_whisper/transcribe.py#L469

It looks like it's discarding the end of the audio, and not transcribing it. Why?

**Comment by trungkienbkhn** [comment, marked as answer]
Same as this [logic](https://github.com/openai/whisper/blob/ba3f3cd54b0e5b8ce1ab3de13e32122d0d5f98ab/whisper/transcribe.py#L134) from original whisper project. You can refer to this [commit](https://github.com/openai/whisper/commit/919a713499d4c4f76ecf448617ada8e9ac008fbb).
